torch_xla/csrc/tensor.cpp

```diff
@@ -1570,7 +1602,12 @@ XLATensor::OpByOpAsync XLATensor::SyncTensorsGraphOpByOp(
                                   &coll.indices);
 
   std::vector<torch::lazy::Value> roots = CollectRoots(*tensors, coll.indices);
-  auto tensors_data = FetchTensorData(tensors, coll.config, coll.indices);
+  std::vector<torch::lazy::Value> ir_values;
+  std::vector<torch::lazy::BackendDataPtr> tensor_data_vec;
+  ExtractIRAndPrepareXlaData(tensors, coll.config, coll.indices, ir_values,
```

Comment:
nit `ExtractIRAndPrepareXlaData_` instead since it will inplace update `ir_values` and `tensor_data_vec`

Reply:
Just curious, what's the reasoning behind `_`, is it for indicating in-place updates ?

Reply:
Oh nvm, I see it!